src/doc/unstable-book/src/compiler-flags/crate-attr.md

```diff
@@ -0,0 +1,8 @@
+# `crate-attr`
+
+The tracking issue for this feature is: [#138287](https://github.com/rust-lang/rust/issues/138287).
+
+------------------------
+
+The `-Z crate-attr` flag allows you to inject attributes into the crate root.
+For example, `-Z crate-attr=crate_name="test"` acts as if `#![crate_name="test"]` were present before the first source line of the crate root.
```

Comment:
What does it look like to have several attrs? `-Zcrate-attr=a=b,c=d`? or do you provide several at once like `-Zcrate-attr=a=b -Zcrate-attr=c=d`? That would be useful to document.

Reply:
GREAT question, apparently it's the latter. documenting that now